elements/powerprediction.lua

```diff
@@ -66,29 +66,26 @@ local function Update(self, event, unit)
 
 	local _, _, _, startTime, endTime, _, _, _, spellID = UnitCastingInfo(unit)
 	local mainPowerType = UnitPowerType(unit)
-	local hasAltManaBar = ALT_MANA_BAR_PAIR_DISPLAY_INFO[playerClass] and ALT_MANA_BAR_PAIR_DISPLAY_INFO[playerClass][mainPowerType]
+	local hasAltManaBar = ALT_MANA_BAR_PAIR_DISPLAY_INFO[playerClass]
```

Comment:
We can upvalue this since the player class can never change.

Reply:
You are late to the party :)

Reply:
We'll get 'em next time

Reply:
I could literally hear that CoD meme in my head :V